readme.md

```diff
@@ -31,14 +31,17 @@ button.addEventListener('click', () => {
 
 ## API
 
-### copy(text)
+### copy(text, options?)
 
 Copy `text` to the clipboard.
 
 Returns a `boolean` of whether it succeeded to copy the text.
 
 Must be called in response to a user gesture event, like `click` or `keyup`.
 
+### Options
+
+- `target`: Optionally specify a DOM element (`document.body` by default) where the temporary, behind-the-scenes `textarea` should be appended. Specify this in cases where you need to stay within a focus trap, like in a modal.
```

Comment:
The readme and index.d.ts docs texts should be in sync. I think the description here is clearer than in index.d.ts.